app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -309,3 +309,27 @@ examples:
             text: How people read
           content:
             html: <p class="govuk-body">This is the content for How people read.</p>
+  with_event_tracking:
+    description: |
+      To switch on Google Analytics for the "Show all sections" button on click, pass `event_tracking: true` the values passed on open will be
+      `Event Action: accordionOpened` `Event Category: pageElementInteraction` `Event Label: Show all sections` 
+    data:
+      event_tracking: true
+      items:
+        - heading:
+            text: Writing well for the web
+            id: writing-well-for-the-web
+          content:
+            html: <p class="govuk-body">This is content for accordion 1 of 2</p><p class="govuk-body">This content contains a <a href="#anchor-nav-test" class="govuk-link">link</a></p>
+        - heading:
+            text: Writing well for specialists
+          content:
+            html: <p class="govuk-body" id="anchor-nav-test">This is content for accordion 2 of 2</p>
+        - heading:
+            text: Know your audience
+          content:
+            html: <p class="govuk-body">This is the content for Know your audience.</p>
+        - heading:
+            text: How people read
+          content:
+            html: <p class="govuk-body">This is the content for How people read.</p>
```

Comment:
This example seems overly long - it only needs to demonstrate the show all button click, two accordion sections. will be enough.